vmm/src/config.rs

```diff
@@ -1515,6 +1512,10 @@ impl NetConfig {
             return Err(ValidationError::NoHardwareChecksumOffload);
         }
 
+        if self.mask.is_some() && self.ip.is_none() {
+            return Err(ValidationError::MaskProvidedWithoutIp);
+        }
```

Comment:
Do you think it also makes sense to require a mask? What does it default to - /32?

Reply:
It defaults to `/8`. In most cases this is probably not useful ;)

Reply:
/8 is definitely something I would not expect (maybe /24 but, not /8). Let's go with option 3 please.